apidoc/v1/index.html

```diff
@@ -215,173 +215,173 @@
         </div>
       <ul id="toc" class="toc-list-h1">
           <li>
-            <a href="#overview" class="toc-h1 toc-link" data-title="Overview">Overview</a>
```

Comment:
Do not change this file. This a autogenerated file from docs/ dir

Reply:
alright doing this now, done for db/schema.rb